### DISCUSSION TITLE
Creating a user from M2M

### DISCUSSION THREAD

**Discussion by sbont** [zitadel/zitadel#5785] [Q&A] [answered]
I'm trying to build a custom registration interface where my server will eventually send a request to the Management API to Create/Import User. However, the listed authentication flow for this endpoint is `authorizationCode`. How do I get this to work in this scenario, where I'm calling from a (trusted) machine? I've also tried basic authentication but that gives me `invalid auth header`. Thanks!

**Comment by hifabienne** [comment, marked as answer]
Hei @sbont 
You can find in your docs guides about how to access our ZITADEL APIs.
Authenticate Service Users: https://zitadel.com/docs/guides/integrate/serviceusers
There are different possibilities JWT Profile, Client Credential and PAT
And how to access the APIs: https://zitadel.com/docs/guides/integrate/access-zitadel-apis
